app/main.py

```diff
@@ -1,8 +1,56 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+        self.wash_ability = None
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float | int,
+                 clean_power: int,
+                 average_rating: float | int,
+                 count_of_ratings: int
+                 ) -> None:
```

Comment:
In case there are no arguments on the same line with a closing parenthesis, your opening parenthesis should not have any arguments on the same line as well. Please, resolve this issue